packages/fxa-auth-db-mysql/lib/db/schema/patch-107-108.sql

```diff
@@ -0,0 +1,49 @@
+-- This migration reverts changes from patch-097-098, removing references
+-- to accountSubscriptions and effectively rolling back to deleteAccount_16
+-- from patch-095-096
+
+SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+
+CALL assertPatchLevel('107');
+
+-- DROP PROCEDURE `createAccountSubscription_1`;
+-- DROP PROCEDURE `deleteAccountSubscription_1`;
+-- DROP PROCEDURE `getAccountSubscription_1`;
+-- DROP PROCEDURE `fetchAccountSubscriptions_1`;
+-- DROP PROCEDURE `deleteAccount_17`;
+-- DROP TABLE accountSubscriptions;
```

Comment:
Are these suppose to be commented out?

Reply:
I'm actually a little confused about this myself, now that you mention it: In forward migrations, we seem to never drop procedures and rarely drop tables. In backward migrations, all of the statements are commented out. I split the difference here and included the drops but commented them out.

Doesn't necessarily hurt anything to leave these procedures and that table lying around, since they won't be used anymore, but would be nice to clean up.

Reply:
I was confused about this too, and wrote myself a note to figure out how our migrations work 😆